src/config/backpack/operations/create.php

```diff
@@ -35,6 +35,9 @@
     // Should we warn a user before leaving the page with unsaved changes?
     'warnBeforeLeaving' => false,
 
+    // Should the create process (queries to insert the entry and it's relationships) be wrapped in a transaction?
+    'useDatabaseTransactions' => false,
```

Comment:
@pxpm shouldn't this be a more broad setting?
Instead of configuring for each operation, using DB Transactions should be something more global, so the developers won't miss changing the setting in some operations.
Using DB Transactions is something that you either fully want or not, you don't want it for a single operation right?

Reply:
Well the thing is ... we don't have it implemented anywhere else. (but we should probably do it in the bulks etc if we are not doing it in a single query). 

So what could happen is that someone create a custom operation and wants to add the option to run that operation queries in a transaction, similar to what we are doing here with create and update. 

If the setting is global, there is no way to keep it for create and update, and disable it in any other operation. Developer would need to create a different setting for the same thing. 

Maybe we mention in both places that there is a "sister" setting for create and update operations?

Or to make it global we change the name to `useDbTransactionsOnCreateAndUpdate` ?  

What you think ?

Reply:
Could we have some kind of inheritance?
Because, in most cases, and this one for sure, I'll want to have DB Transactions everywhere. I can't see a situation where one would want everywhere but one place.
If we had some kind of inheritance it would solve that problem. By default it's true in `operations.php` config, but in some specific edge case operation I could disable it 🤷‍♂️
Is this doable?

Reply:
I think I get it, you mean **global** true, and check **per** operation if it's overriden ? 

Something like:
```php
//config/operations.php
`useDatabaseTransactions` => true/false

// config/create.php (optional)
'useDatabaseTransactions` => true/false
```
and in code we use something like:
```php
config('create.useDatabaseTransactions') ?? config('operations.useDatabaseTransactions');
```

That way you only need to enable it once, but you can overwrite it per operation/crud etc.

Will do the change. 

Thanks